modules/sdk-coin-avaxp/src/lib/transferBuilder.ts

```diff
@@ -10,7 +10,7 @@ export class TransferBuilder extends TransactionBuilder {
   }
 
   protected get transactionType(): TransactionType {
-    return TransactionType.WalletInitialization;
+    return TransactionType.Send;
```

Comment:
there's no send in avaxp ?

Reply:
The transferBuilder is there. I fix its transactionType, Should I remove it directly?